e2e/support/helpers/e2e-dashboard-helpers.ts

```diff
@@ -296,7 +296,7 @@ export function resizeDashboardCard({
 
 /** Opens the dashboard info sidesheet */
 export function openDashboardInfoSidebar() {
-  dashboardHeader().icon("info").click();
+  dashboardHeader().findByLabelText("More info").should("be.enabled").click();
```

Comment:
`.should("be.enabled")` is not really needed. `.click()` does that for you. it the test flakes without it, `.findByLabelText()` is not returning the proper element. But that I’d say is unlikely unless there some weird element wrapping situation going on